### DISCUSSION TITLE
Terraform.tfvars is automatically loaded

### DISCUSSION THREAD

**Discussion by gaurav9009** [learndevopsonline/learndevopsonline#1865] [Q&A]
https://github.com/raghudevopsb78/learn-terraform/blob/main/03-variables/var.tf#L52

I presume in the above file on line 52, instead of "Terraform.auto.tfvars" it should be "Terraform.tfvars". 
Also "*.auto.tfvars" takes precedence over "Terraform.tfvars" file if the same variable is mentioned in both

**Comment by chaithanya1812** [comment]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/8dfcc630-c58b-4a11-8d97-e87a6df658f1)
